addons/bestja_frontend_ucw/views/home_page.xml

```diff
@@ -302,107 +301,51 @@
-                                                    </div>
-                                                </figcaption>
-                                            </figure>
-                                            <figure>
-                                                <a class="photostack-img">
-                                                    <img src="http://lorempixel.com/g/350/350" alt="img07"/>
-                                                </a>
-                                                <figcaption>
-                                                    <p class="photostack-title">Zdjęcie 7</p>
-                                                    <div class="photostack-back">
-                                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
-                                                    </div>
-                                                </figcaption>
-                                            </figure>
-                                        </div>
-                                        <script src="/bestja_frontend_ucw/static/src/js/modernizr.min.js"></script>
-                                        <script src="/bestja_frontend_ucw/static/src/js/classie.js"></script>
-                                        <script src="/bestja_frontend_ucw/static/src/js/photostack.js"></script>
-                                        <script>
-                                            // [].slice.call( document.querySelectorAll( '.photostack' ) ).forEach( function( el ) { new Photostack( el ); } );
-
-                                            new Photostack( document.getElementById( 'photostack-1' ), {
-                                            callback : function( item ) {
-                                            }
-                                            } );
-                                        </script>
+                                        </t>
                                     </div>
+                                    <script src="/bestja_frontend_ucw/static/src/js/modernizr.min.js"></script>
+                                    <script src="/bestja_frontend_ucw/static/src/js/classie.js"></script>
+                                    <script src="/bestja_frontend_ucw/static/src/js/photostack.js"></script>
+                                    <script>
+                                        // [].slice.call( document.querySelectorAll( '.photostack' ) ).forEach( function( el ) { new Photostack( el ); } );
```

Comment:
It has been removed, long time ago commented and unnecessary now. Photostack is working without this code.